packages/financial-templates-lib/test/clients/ExpiringMultiPartyClient.js

```diff
@@ -12,7 +12,7 @@ const {
 const { getTruffleContract } = require("@uma/core");
```

Comment:
Are you planning to change the name of this file?

Reply:
yup, good catch.